iFixit/react-commerce
# Add OEM badge

connects #797 

<img width="537" alt="Screenshot 2022-10-27 at 16 33 11" src="https://user-images.githubusercontent.com/7805759/198315921-cde7124f-570e-4560-a901-0e2bd5074e40.png">

### QA

1. Visit Vercel preview
2. Try modifying the OEM Partnership JSON metafield value associated with the product to verify that the banner is working as expected in all scenarios
